### PR TITLE
Very small (mostly) improvements

### PR DESCRIPTION
**Pull request by mcfnord**
**Short description of changes**
Tiny improvements just to give it a try.

**Context: Fixes an issue? Related issues**
Improved one page.

**Status of this Pull Request**

**What is missing until this pull request can be merged?**

**Does this need translation?**

## Checklist

- [x] I've verified that this Pull Request follows the [general code principles](https://github.com/jamulussoftware/jamulus/blob/master/CONTRIBUTING.md#jamulus-projectsource-code-general-principles)
- [ ] I waited some time after this Pull Request was opened and all GitHub checks completed without errors. 
- [ ] I'm sure that this Pull Request goes to the correct branch

### CODE REVIEW

**Comment by ignotus666** [comment]
Hi! Thanks for contributing.

Since these are changes that affect translations, the PR has to be made against the 'next-release' branch (I changed it).

File: wiki/en/Getting-Started.md [hunk 1]
```diff
@@ -13,28 +13,28 @@ To get the best from Jamulus, at a minimum you will need:
 1. **Wired headphones** (not Bluetooth or speakers - see the [FAQ](/wiki/FAQ#why-shouldnt-i-use-wireless-equipment))
 1. **A reasonable audio device, sound card and/or mic** ([see this list](/kb/2021/01/05/Jamulus-Sound-Devices.html) for examples)
 
-If you have any questions, please see the [Discussions](https://github.com/jamulussoftware/jamulus/discussions)
+Questions? See the [Discussions](https://github.com/jamulussoftware/jamulus/discussions)
```
Comment (ignotus666) [inline wiki/en/Getting-Started.md:16]: While I can see the rationale behind these changes (economy of style), we have to bear in mind that even the smallest edits mean that all translations have to be updated - even when the change is so subtle that the translation will stay the same. Does the change clarify/correct/improve the text enough to warrant this?

File: wiki/en/Getting-Started.md [hunk 2]
```diff
@@ -13,28 +13,28 @@ To get the best from Jamulus, at a minimum you will need:
...
 
 ## Installation
 
-To get Jamulus working with your operating system, please follow the installation guide for your platform:
+To get Jamulus working with your operating system, follow the installation guide for your platform:
```
Comment (ignotus666) [inline wiki/en/Getting-Started.md:20]: See above.

File: wiki/en/Getting-Started.md [hunk 3]
```diff
@@ -13,28 +13,28 @@ To get the best from Jamulus, at a minimum you will need:
...
 
 <div class="fx-row fx-row-start-xs button-container">
     <a href="Installation-for-Windows" class="button fx-col-100-xs">Windows</a>
     <a href="Installation-for-Macintosh" class="button fx-col-100-xs">macOS</a>
     <a href= "Installation-for-Linux" class="button fx-col-100-xs">Linux</a>
 </div>
-_[Android](Installation-for-Android) and [iOS](Installation-for-iOS) can be used too, but are still considered experimental._
+_[Android](Installation-for-Android) and [iOS](Installation-for-iOS) can be used too, but are still experimental._
```
Comment (ignotus666) [inline wiki/en/Getting-Started.md:27]: See above.

File: wiki/en/Getting-Started.md [hunk 4]
```diff
@@ -13,28 +13,28 @@ To get the best from Jamulus, at a minimum you will need:
...
 
-And for now, close all other applications. It’s best to start simple at first.
+For now, close all other applications. It’s best to start simple at first.
```
Comment (ignotus666) [inline wiki/en/Getting-Started.md:29]: See above.

File: wiki/en/Getting-Started.md [hunk 5]
```diff
@@ -13,28 +13,28 @@ To get the best from Jamulus, at a minimum you will need:
...
 
 ## Hardware Setup
 
-If you use external audio hardware, plug that in before you start Jamulus. If you haven't configured your hardware, please see the installation guides mentioned above.
+If you use external audio hardware, plug it in before you start Jamulus. If you haven't configured your hardware, see the installation guides mentioned above.
```
Comment (ignotus666) [inline wiki/en/Getting-Started.md:33]: See above.

File: wiki/en/Getting-Started.md [hunk 6]
```diff
@@ -13,28 +13,28 @@ To get the best from Jamulus, at a minimum you will need:
...
 
 ## Jamulus main window
 
-When you open Jamulus, you will see a window which looks like this:
+When you open Jamulus, you will see a window that looks like this:
```
Comment (ignotus666) [inline wiki/en/Getting-Started.md:37]: This does correct a mistake.

File: wiki/en/Getting-Started.md [hunk 7]
```diff
@@ -60,7 +60,7 @@ Before you connect to a Server, you should **not** be able to hear yourself.  Li
 
 **Before playing with others, we recommend that you connect to an empty Server to test your setup**, and make sure you are listening to the signal coming back from the Server (if possible) and not yourself.
 
-Now use the “Connect” button in the Jamulus main window to join a Server. A window will now open:
+Now use the “Connect” button in the Jamulus main window to join a Server. A window will open:
```
Comment (ignotus666) [inline wiki/en/Getting-Started.md:63]: See 1st comment.

File: wiki/en/Getting-Started.md [hunk 8]
```diff
@@ -69,7 +69,7 @@ Now use the “Connect” button in the Jamulus main window to join a Server. A
 
 The most important thing about Servers is their “ping time”. The bigger the number, the harder it will be to play in time with others. Usually, you would select a Server with a ping of less than 50ms if you can.
 
-Once connected to a Server, make sure you can hear yourself OK and fix any input volume or other problems. You can have a look at the [troubleshooting page](/wiki/Client-Troubleshooting) for common problems. And of course, check that you are also able to obey [The Golden Rule](/wiki/Client-Troubleshooting#you-all-sound-ok-but-its-difficult-to-keep-together).
+Once connected to a Server, make sure you can hear yourself OK and fix any input volume or other problems. You can have a look at the [troubleshooting page](/wiki/Client-Troubleshooting) for common problems. Also check that you are also able to obey [The Golden Rule](/wiki/Client-Troubleshooting#you-all-sound-ok-but-its-difficult-to-keep-together).
```
Comment (ignotus666) [inline wiki/en/Getting-Started.md:72]: See 1st comment. Also, a bit of a redundant use of "also" ;)

File: wiki/en/misc/1-index.html [hunk 9]
```diff
@@ -31,8 +31,10 @@ <h2>What is Jamulus?</h2>
 <div class="fx-row fx-row-center-xs">
   <div class="fx-col-100-xs fx-col-50-l">
      <h2>Jamulus worldwide</h2>
-    All over the world Jamulus allows huge choirs to rehearse, loud rock bands to play, and brings folk and classical musicians together. It's being used for remote music lessons,
-    in schools and universities, in private and in public. Even just for the spoken word – all in real time on the Internet, as if you were there in person.
+    All over the world Jamulus allows huge choirs to rehearse, and loud rock bands to play. Jamulus brings folk and classical musicians together.
+    It's being used for remote music lessons,
+    in schools and universities, in private and in public. Even just for the spoken word. All in real time on the Internet, as if you were together in one room.
```
Comment (ignotus666) [inline wiki/en/misc/1-index.html:36]: Not sure I prefer the staccato style over the original.

**Comment by ignotus666** [comment]
I have nothing against changing text if it is an improvement (and I do think some of these changes improve readability), but all in all I don't think these changes are worth the effort they would require from other people.

File: _posts/2020-03-28-Server-Rpi.md [hunk 10]
```diff
@@ -11,15 +11,15 @@ This guide shows you how to install Jamulus on a Raspberry Pi.
 
 ## Server Installation - Raspberry Pi
 
-We now recommend you use the official armhf .deb files ([GUI download]({{ site.download_root_link }}{{ site.download_file_names.deb-gui-armhf }}) [headless download]({{ site.download_root_link }}{{ site.download_file_names.deb-headless-armhf }})).  However, you can still compile Jamulus for your Pi from source, if you wish.
+We recommend you use the official armhf .deb files ([GUI download]({{ site.download_root_link }}{{ site.download_file_names.deb-gui-armhf }}) or [headless download]({{ site.download_root_link }}{{ site.download_file_names.deb-headless-armhf }})).  However, you can still compile Jamulus for your Pi from source, if you wish.
```
Comment (pljones) [inline _posts/2020-03-28-Server-Rpi.md:14]: This _needs_ "or" between "GUI download" and "headless download" or it read, wrongly, "[GUI download](.)  [headless download](.)".

File: wiki/en/Getting-Started.md [hunk 11]
```diff
@@ -69,7 +69,7 @@ Now use the “Connect” button in the Jamulus main window to join a Server. A
...
 
 ## Playing for the first time
```
Comment (pljones) [inline wiki/en/Getting-Started.md:75]: There's only one change I'd agree with in all of this file and it's not worth doing.

File: wiki/en/misc/1-index.html [hunk 12]
```diff
@@ -31,8 +31,10 @@ <h2>What is Jamulus?</h2>
...
+    
     <h2>Help needed?</h2>
```
Comment (pljones) [inline wiki/en/misc/1-index.html:38]: Pointless line break.

**Review by pljones — CHANGES_REQUESTED** [review]
Please don't request changes without good reason.

File: wiki/en/Getting-Started.md (hunk 1 above)
Comment (mcfnord) [reply wiki/en/Getting-Started.md:16]: Is there a good way to introduce changes like this one? _En masse_ somehow? Can just the English publication contain these kinds of changes even if they aren't translated?

File: wiki/en/Getting-Started.md (hunk 6 above)
Comment (mcfnord) [reply wiki/en/Getting-Started.md:37]: Though perhaps an inconsequential one.

File: wiki/en/Getting-Started.md (hunk 11 above)
Comment (mcfnord) [reply wiki/en/Getting-Started.md:75]: If you mean https://github.com/jamulussoftware/jamuluswebsite/pull/827/files/b0789fbad59a2f0e184ea6574054ba724f89d2b5#r977239188...

1) Why would a PR that _only_ added **or** not be worth doing?
2) Is there a scenario where **or** could be added?

**Comment by ann0see** [comment]
@mcfnord first of all thank you very much for your contribution. In general, I'd say they're worth adding. 

@ignotus666 I think it's possible to edit the .po files in batch via search and replace.

**Comment by ignotus666** [comment]
> @ignotus666 I think it's possible to edit the .po files in batch via search and replace.

But why would we need to edit .po files like that? If the EN texts change and we want those changes reflected in translations it should be done by translators. Or if the changes in EN are so small that translations will remain the same, they will need to be reverted from "fuzzy" to "confirmed".

**Comment by mcfnord** [comment]
@ignotus666 I could tag PRs where the underlying meaning does not change, and that will be typical, as mostly I improve clarity rather than change meaning.

**Pull request closed without merging**